test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java

```diff
@@ -226,8 +225,7 @@ private Process startSegmentStore(int segmentStoreId) throws IOException {
                 .sysProp(configProperty(ServiceConfig.COMPONENT_CODE, ServiceConfig.CERT_FILE), "../../config/cert.pem")
                 .sysProp(configProperty(BookKeeperConfig.COMPONENT_CODE, BookKeeperConfig.ZK_ADDRESS), getZkUrl())
                 .sysProp(configProperty(ServiceConfig.COMPONENT_CODE, ServiceConfig.LISTENING_PORT), port)
-                .sysProp(configProperty(ServiceConfig.COMPONENT_CODE, ServiceConfig.STORAGE_IMPLEMENTATION), ServiceConfig.StorageType.FILESYSTEM)
-                .sysProp(configProperty(FileSystemStorageConfig.COMPONENT_CODE, FileSystemStorageConfig.ROOT), getSegmentStoreStoragePath())
+                .sysProp(configProperty(ServiceConfig.COMPONENT_CODE, ServiceConfig.STORAGE_IMPLEMENTATION), ServiceConfig.StorageType.INMEMORY)
```

Comment:
This is a change of behavior. Why are you suggesting we change this on selftest?

Reply:
As we are not building the bindings, the implementation for `FILESYSTEM` will not be loaded in travis. When this process is run, whether we write to local filesystem OR to local mem does not make any difference as both are not real deployments.